docs/projects/earm-pipeline/about.md

```diff
@@ -0,0 +1,41 @@
+---
+id: about
+title: eARM Scientific Workflow with RAG & HPC Deployment
+custom_edit_url: null
+---
+
+## Overview
+
+Adapting and contributing to the open-source eARM molecular modeling workflow by integrating Retrieval-Augmented Generation (RAG) knowledge retrieval capabilities and deploying the scientific workflow on SLU HPC systems.
```

Comment:
I like that you are thinking about both projects. When we think about them as projects, though, let's make sure we keep them as two distinct entities. In my mind, it's "The HPC Team" and the team is starting on the "eARM HPC Workflow", and then eventually will start working on the "HPC RAG agent". Please, tell me if I'm missing something, though. You've been talking much more directly and more recently with Dr. Donlin. Do we have a RAG requirement in the eARM project, too?